pixeltable/dataframe.py

```diff
@@ -848,6 +852,29 @@ def group_by(self, *grouping_items: Any) -> DataFrame:
             limit=self.limit_val,
         )
 
+    def distinct(self) -> DataFrame:
+        """
+        Remove duplicate rows from this DataFrame.
+
+        Note that grouping will be applied to the rows based on the select clause of this Dataframe.
+        In the absence of a select clause, by default, all columns are selected in the grouping.
+
+        Examples:
+            Get all unqiue addresses from table `addresses`.
+
+            >>> results = addresses.distinct()
```

Comment:
distinct() doesn't (and shouldn't) call collect(), so this examples is probably misleading.